cask_index.html

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+permalink: /cask/
+---
+<p>This is a listing of all Casks available via the <a href="https://brew.sh">Homebrew</a> package manager for macOS.</p>
+
+<h2><a href="{{ site.baseurl }}/api/cask.json"><code>/api/cask.json</code> (JSON API)</a></h2>
+
+<table>
+    {%- assign sorted_casks = site.data.cask | sort -%}
+    {%- for cask in sorted_casks -%}
+
+    <tr>
+        {%- assign t = cask[0] -%}
+        {%- assign c = cask[1] -%}
+        {%- include cask.html token=t cask=c -%}
```

Comment:
Is it worth including both the Cask token and the Cask name? They are very similar. May be worth having the version follow the name like for formulae.

Reply:
The cask token has to be passed to `_includes/cask.html` because, unlike formulae, it's not actually included within the `cask[1]` data structure.

Reply:
Sorry, to be explicit: I mean is it worth outputting them both on the index page in separate columns?

Reply:
1. search engines: since cask tokens don't always match their names letter-for-letter, including both strengthens the association for when search engines crawl the site.
2. cask discoverability: when viewing the list of casks, including the cask name allows a user doing an in-page search of, say, "PyCharm Community Edition" to jump to the `pycharm-ce` row.

Reply:
Works for me 👍